cli/src/main/scala/co/uproot/abandon/App.scala

```diff
@@ -107,25 +110,33 @@ object CLIMain  {
 
     reportWriter.println("Account Name: " + bookReportSettings.account + "\n")
 
-    val maxNameLength = maxElseZero(bookReport.flatMap(_.entries.flatMap(_.txns.flatMap(_.parentOpt.get.children.map(_.name.fullPathStr.length)))))
+    val maxNameLength = maxElseZero(bookReport.flatMap(_.entries.flatMap(_.txns.flatMap(_.parentOpt.fold {
+      ??? // What do we do in the event we have no parent?
+    } { parent =>
+      parent.children.map(_.name.fullPathStr.length)
+    }))))
```

Comment:
It would be nicer if you convert the `Option.get` into `Option match {case ....}`, rather than `Option.fold`.